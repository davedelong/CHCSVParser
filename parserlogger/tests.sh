#!/bin/sh

#  rts.sh
#  CHCSVParser
#
#  Created by Malte Tancred on 2013-05-10.
#

export PATH="$PATH":"$BUILT_PRODUCTS_DIR"
exec 2>&1

echo '---test empty'
parserlogger ''
echo $?

echo '---test one field'
parserlogger 'field'
echo $?

echo '---test two fields'
parserlogger 'field1,field2'
echo $?
