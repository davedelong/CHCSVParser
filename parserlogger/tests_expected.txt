---test empty
BEGIN DOCUMENT
BEGIN LINE 1
FIELD 0: ''
END LINE 1
END DOCUMENT
0
---test one field
BEGIN DOCUMENT
BEGIN LINE 1
FIELD 0: 'field'
END LINE 1
END DOCUMENT
0
---test two fields
BEGIN DOCUMENT
BEGIN LINE 1
FIELD 0: 'field1'
FIELD 1: 'field2'
END LINE 1
END DOCUMENT
0
